docs/troubleshooting.rst

```diff
@@ -193,3 +193,16 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+fatal: the remote end hung up unexpectedly / fatal: early EOF / fatal: index-pack failed when running `tutor images build ...`
```

Comment:
In RST, verbatim strings are surrounded by double ticks: ``` ``tutor images build`` ```

Reply:
Done. Thanks for letting me know!